### PR TITLE
Install sqlsrv on PHP 8.1

### PR DESCRIPTION
**Pull request by ddeboer**

### CODE REVIEW

File: layers/blackfire/config.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "php": [
         "73",
         "74",
-        "80"
+        "80",
+        "81"
```
Comment (GrahamCampbell) [inline layers/blackfire/config.json:6]: This change is not related, and should be in a different PR

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: layers/blackfire/config.json (hunk 1 above)
Comment (ddeboer) [reply layers/blackfire/config.json]: Oops, sorry, that shouldn’t be there. Removed.

**Comment by Nyholm** [comment]
Thank you. 

Could we do a workaround so the beta release is not installed on php 8.0 and lower?

**Comment by ddeboer** [comment]
@Nyholm Done!

**Review by Nyholm — APPROVED** [review]
Thank you
